### DISCUSSION TITLE
Hot reload

### DISCUSSION THREAD

**Discussion by goldenpath-ai** [directus/directus#16156] [Feature Requests]
Hello,

Is it possible to achieve hot reloading on custom modules? I know about the env variable that rebuilds the extension but reloading the browser as well and not having to refresh every single time I update something would be nice.

**Comment by br41nslug** [comment]
https://github.com/directus/directus/discussions/8190
https://github.com/directus/directus/discussions/3044
https://github.com/directus/directus/discussions/5386

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
